content/v1.0.0/index.md

```diff
@@ -0,0 +1,200 @@
+---
+draft: false
+aliases: ["/en/"]
+---
+
+# Conventional Commits 1.0.0
+
+## Summary
+
+The Conventional Commits specification is a lightweight convention on top of commit messages.
+It provides an easy set of rules for creating an explicit commit history;
+which makes it easier to write automated tools on top of.
+This convention dovetails with [SemVer](http://semver.org),
+by describing the features, fixes, and breaking changes made in commit messages.
+
+The commit message should be structured as follows:
+
+---
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer]
+```
+---
+
+<br />
+The commit contains the following structural elements, to communicate intent to the
+consumers of your library:
+
+1. **fix:** a commit of the _type_ `fix` patches a bug in your codebase (this correlates with [`PATCH`](http://semver.org/#summary) in semantic versioning).
+1. **feat:** a commit of the _type_ `feat` introduces a new feature to the codebase (this correlates with [`MINOR`](http://semver.org/#summary) in semantic versioning).
+1. **BREAKING CHANGE:** a commit that has the text `BREAKING CHANGE:` at the beginning of its optional body or footer section introduces a breaking API change (correlating with [`MAJOR`](http://semver.org/#summary) in semantic versioning).
```

Comment:
I've been using `fix!: this is my breaking change` much more than `BREAKING CHANGE:` these days.

I'm thinking in this quick start section we should perhaps suggest this approach rather than having `BREAKING CHANGE:` in the body.

Reply:
I think we can add some words to at least say that `!` is not a thing.

But I would not suggest which one to use because the `BREAKING CHANGE` is still mandatory.

Reply:
I think it should one or the other or both.

Reply:
Breaking changes MUST be described, but body is optional, therefore `BREAKING CHANGES:` in body shouldn't be mandatory. If there is no body or body doesn't have `BREAKING CHANGES: ` and theres is `!` before `:` then description from summary line should be treated as description of breaking changes. And if there is `BREAKING CHANGES:` in the body that should be treated as description of breaking changes.

Reply:
`BREAKING CHANGE` is already not being treated as mandatory by the tooling I maintain; I find it much more convenient to use `!` most of of the time -- as the description tends to usually call out why the commit was a breaking change.

In recent edits to the spec, I've updated the language to better reflect this; I've also updated the language to describe `BREAKING CHANGE: longer description` as a footer (which seems less ambiguous).

```
feat: my breaking fhange

BREAKING CHANGE: this is a longer description of my
breaking change
```

would, in the updates to the spec, have the footer `BREAKING CHANGE` equal to `this is a longer description of my breaking change`.